src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+function moveColumns() {
+  const table = document.querySelector('.table');
```

Comment:
```suggestion
  const table = document.querySelector('table');
```
here is the problem with failed tests.
In DOM no exists element with class table, but exiests tag table.